### PR TITLE
LPS-153304 Limit characters in Save Fragment Composition's inputs and add description field validation

### PR DESCRIPTION
**Pull request by DiegoHu97**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Comment by ruben-pulido** [comment]
Just started reviewing :)

File: modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/service/impl/FragmentCompositionLocalServiceImpl.java [hunk 1]
```diff
@@ -74,6 +75,7 @@ public FragmentComposition addFragmentComposition(
 		fragmentCompositionKey = _getFragmentCompositionKey(
 			fragmentCompositionKey);
 
+		validateDescription(description);
```
Comment (ruben-pulido) [inline modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/service/impl/FragmentCompositionLocalServiceImpl.java:78]: This should be sorted following the order the fields have been defined in service.xml. As an example you can see the order used when setting name and description

File: modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/service/impl/FragmentCompositionLocalServiceImpl.java [hunk 2]
```diff
@@ -292,6 +294,7 @@ public FragmentComposition updateFragmentComposition(
 			fragmentCompositionPersistence.findByPrimaryKey(
 				fragmentCompositionId);
 
+		validateDescription(description);
```
Comment (ruben-pulido) [inline modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/service/impl/FragmentCompositionLocalServiceImpl.java:297]: Sorting

File: modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorDisplayContext.java [hunk 3]
```diff
@@ -424,6 +425,14 @@ public Map<String, Object> getEditorContext(String npmResolvedPackageName)
 			).put(
 				"getFormFieldsURL",
 				_getResourceURL("/layout_content_page_editor/get_form_fields")
+			).put(
+				"getFragmentCompositionDescriptionMaxLength",
```
Comment (ruben-pulido) [inline modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorDisplayContext.java:429]: I would name these fragmentCompositionDescriptionMaxLength and fragmentCompositionNameMaxLength

**Comment by DiegoHu97** [comment]
Resent here: https://github.com/liferay-echo/liferay-portal/pull/8421

**Pull request closed without merging**